README.md

```diff
@@ -39,7 +39,7 @@ When you're a first semester freshman, it's okay to have old high school stuff o
 ### Don't make it a scavenger hunt.
 When an application reviewer (engineer, recruiter, or otherwise) is looking over your resume, don't make it difficult for them to understand who you are and what you know. 
 
-For example, if you have online profiles like GitHub, LinkedIn, Twitter, or even your own personal website, put it on your resume.  Don't make them have to search long and hard for you online if they want to know more!
+For example, if you have online profiles like GitHub, LinkedIn, Twitter, or even your own personal website, put it on your resume.  Don't make them have to search long and hard for you online if they want to know more! Make sure the URLs are both easy to remember and convenient to type. Having a long, alphanumeric URL is not going to help your case. LinkedIn allows you to create a shorter link to your LinkedIn profile and you can use either tiny.cc or bit.ly to shorten links to portfolios.
```

Comment:
Hi Cassidy,

What do you think of these changes?

https://github.com/dsouzarc/getting-a-gig/compare/cassidoo:master...master


On Sun, Jan 11, 2015 at 10:30 PM, Ryan Dsouza <dsouzarc@gmail.com> wrote:

> Ok. But what about the rest of the changes? I added a small portion about
> making sure to practice without a computer/fanciful IDE because you're not
> going to have autocomplete or compiler checking during an interview.
>
> There's also the amendment about the different what questions (What was
> the most difficult part about this project? what could you have done
> better?)
>
>
> On Sunday, January 11, 2015, Cassidy Williams <notifications@github.com>
> wrote:
>
>> In README.md
>> <https://github.com/cassidoo/getting-a-gig/pull/10#discussion-diff-22773982>
>> :
>>
>> > @@ -39,7 +39,7 @@ When you're a first semester freshman, it's okay to have old high school stuff o
>> >  ### Don't make it a scavenger hunt.
>> >  When an application reviewer (engineer, recruiter, or otherwise) is looking over your resume, don't make it difficult for them to understand who you are and what you know.
>> >
>> > -For example, if you have online profiles like GitHub, LinkedIn, Twitter, or even your own personal website, put it on your resume.  Don't make them have to search long and hard for you online if they want to know more!
>> > +For example, if you have online profiles like GitHub, LinkedIn, Twitter, or even your own personal website, put it on your resume.  Don't make them have to search long and hard for you online if they want to know more! Make sure the URLs are both easy to remember and convenient to type. Having a long, alphanumeric URL is not going to help your case. LinkedIn allows you to create a shorter link to your LinkedIn profile and you can use either tiny.cc or bit.ly to shorten links to portfolios.
>>
>> I'm not sure if this is actually that important. Usually resume parsers
>> just export the URL from the, and interviewers can just click on it from
>> there.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/cassidoo/getting-a-gig/pull/10/files#r22773982>.
>>
>